solr/core/src/java/org/apache/solr/rest/ManagedResourceStorage.java

```diff
@@ -421,8 +421,20 @@ public String getInfo() {
   /** Default storage implementation that uses JSON as the storage format for managed data. */
   public static class JsonStorage extends ManagedResourceStorage {
 
+    private final int indentSize;
+
+    /** Uses 2 space characters as an indent. */
     public JsonStorage(StorageIO storageIO, SolrResourceLoader loader) {
+      this(storageIO, loader, 2);
```

Comment:
if this '2' is a default parameter I would prefer to see it as a constant, it would be more readable

Reply:
Furthermore, who's calling this default?

Reply:
> if this '2' is a default parameter I would prefer to see it as a constant, it would be more readable

https://github.com/apache/solr/pull/2030/commits/cc4a2d44abfbdc4f151aed39b18f60769cb1f11e creates a `JSONWriter.DEFAULT_INDENT` constant and uses it here.

Though I could also see that perhaps that's undesirable since it exposes `JSONWriter` implementation detail here i.e. a constant elsewhere would avoid that.

Reply:
Took a look at that commit, and it looks good to me! I don't see much problem with it!